l10n_fi_payment_terms/models/payment_term.py

```diff
@@ -0,0 +1,38 @@
+# -*- coding: utf-8 -*-
+
+# 1. Standard library imports:
+
+# 2. Known third party imports:
+
+# 3. Odoo imports (openerp):
+from openerp import api, fields, models
+
+# 4. Imports from Odoo modules:
+
+# 5. Local imports in the relative form:
+
+# 6. Unknown third party imports:
+
+
+class PaymentTerm(models.Model):
+
+    # 1. Private attributes
+    _inherit = 'account.payment.term'
+
+    # 2. Fields declaration
+    _order = 'sequence, name'
+
+    sequence = fields.Integer('Sequence')
+    code = fields.Char('Unique code')
```

Comment:
I think these aren't absolutely necessary, but they are useful as alphabetical ordering is confusing in this instance. They should, however, be introduces in another module than a localization one.